pkg/kubelet/kubelet_node_status.go

```diff
@@ -315,7 +315,7 @@ func (kl *Kubelet) initialNode() (*v1.Node, error) {
 // It synchronizes node status to master, registering the kubelet first if
 // necessary.
 func (kl *Kubelet) syncNodeStatus() {
-	if kl.kubeClient == nil {
+	if kl.kubeClient == nil || kl.standaloneMode {
```

Comment:
Done, I also made sure that when in standalone mode, the clients are set to nil, so that things are consistent.